api/src/pcapi/core/finance/api.py

```diff
@@ -1401,6 +1402,9 @@ def _generate_cashflows(batch: models.CashflowBatch) -> None:
                 models.Pricing.collectiveBookingId.is_not(None),
                 educational_models.CollectiveStock.beginningDatetime < batch.cutoff,
             ),
+            sqla.and_(
+                models.FinanceEvent.bookingFinanceIncidentId.is_not(None),
+            ),
         ),
     )
```

Comment:
[détail / style] Puisqu'il n'y a plus qu'une seule clause dans le `and`, il n'est plus nécessaire : 

```suggestion
            models.FinanceEvent.bookingFinanceIncidentId.is_not(None),
        ),
    )
```